### PR TITLE
implement gvel collateral health check logic command

### PR DESCRIPTION
**Pull request by athiwatp**

### CODE REVIEW

File: go/cmd/gvel/layers/commands/collateral/collateral_health_check_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+func TestCommandHandler_CollateralHealthCheck(t *testing.T) {
+
+	var (
+		collateralAssetCode    = "VELO"
+		collateralAssetAddress = "0x04"
+		presentAmount          = "1.0000000"
+		requiredAmount         = "2.0000000"
+	)
+
+	t.Run("success", func(t *testing.T) {
```
Comment (Kafakk) [inline go/cmd/gvel/layers/commands/collateral/collateral_health_check_test.go:21]: add multi collateral success case

File: go/cmd/gvel/entity/collateral.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+// CollateralHealthCheck
+type CollateralHealthCheckInput struct {
+	Passphrase string
+}
+
+type CollateralHealthCheckOutput struct {
+	CollateralAssetAddress string
+	CollateralAssetCode    string
+	RequiredAmount         string
```
Comment (Kafakk) [inline go/cmd/gvel/entity/collateral.go]: RequiredCollateralAmount

File: go/cmd/gvel/entity/collateral.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	PresentAmount          string
```
Comment (Kafakk) [inline go/cmd/gvel/entity/collateral.go]: CollateralPoolAmount

File: go/cmd/gvel/layers/commands/collateral/main.go [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+		collateralCommand.GetHealthCheckCommand(),
+	)
+
+	return command
+}
+
+func (collateralCommand *CommandHandler) GetHealthCheckCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   constants.CmdCollateralHealthCheck,
+		Short: "Get collateral health check of Velo",
```
Comment (Kafakk) [inline go/cmd/gvel/layers/commands/collateral/main.go]:
```suggestion
		Short: "Get collateral health check",
```

File: go/cmd/gvel/layers/logic/collateral_health_check.go [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+import (
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/velo-protocol/DRSv2/go/cmd/gvel/entity"
+	"github.com/velo-protocol/DRSv2/go/cmd/gvel/utils/crypto"
+	"github.com/velo-protocol/DRSv2/go/libs/vclient"
+)
+
+func (lo *logic) CollateralHealthCheck(input *entity.CollateralHealthCheckInput) ([]entity.CollateralHealthCheckOutput, error) {
```
Comment (jura-b) [inline go/cmd/gvel/layers/logic/collateral_health_check.go]: []*entity.CollateralHealthCheckOutput

File: go/cmd/gvel/layers/logic/interface.go [hunk 6]
```diff
@@ -18,6 +18,9 @@ type Logic interface {
 	MintCreditByCollateral(input *entity.MintCreditByCollateralInput) (*entity.MintCreditByCollateralOutput, error)
 	GetCreditExchange(input *entity.GetCreditExchangeInput) (*entity.GetCreditExchangeOutput, error)
 
+	// Collateral
+	CollateralHealthCheck(input *entity.CollateralHealthCheckInput) ([]entity.CollateralHealthCheckOutput, error)
```
Comment (jura-b) [inline go/cmd/gvel/layers/logic/interface.go]: []*entity.CollateralHealthCheckOutput

**Pull request merged**